client/src/app/notice/notice.component.html

```diff
@@ -0,0 +1,9 @@
+<nz-alert
+  nzType="error"
+  nzCloseable
+  nzMessage="This site is in BETA"
+  nzDescription="A DPS sim for SCH based on the original Google sheets sim is available: change stat values, party comp, and rotation info to see the effect on DPS.
+  Changes you make will NOT persist if you close the browser tab.
+  Future updates will allow gearsets to be saved/loaded, custom rotations, support for additional jobs, and more.
+  If you notice any issues or have feedback, please use the button in the lower left to visit us on GitHub, check to see if your issue is already tracked, and file one if it's not. Thanks!"
+></nz-alert>
```

Comment:
Looks like nzDescription either takes a string (that can't be formatted), or a TemplateRef (which is just a section of HTML).

Try putting this after the </nz-alert>:
<ng-template #alertDescription>
  <!-- put your formatted description here -->
</ng-template>

And change the nzDescription to [nzDescription]="alertDescription"

Hopefully that works?

Reply:
yep, it does! thanks!